File: C_Workspace/C_CheckList/Digits_Manipulation/2.c
//C Program to calculate SUM & PRODUCT of all digits.
#include <stdio.h>

int main() 
{
    int n;
    printf("Enter the number: ");
    scanf("%d",&n);
    int m=n;
    int sum,pro=1,rem;
    sum=rem=0;
    while(n>0)
    {
        rem=n%10;
        sum=sum+rem;
        if(rem!=0)
        {
            pro=pro*rem;
        }
        n=n/10;
    }
    printf("The Sum of %d is %d\nThe Product of %d is %d\n",m,sum,m,pro);
    return 0;
}

File: Python_Workspace/Python_Tasks/16_05/2.py
#Harshad number
a=int(input("Enter the number:"))
b=a
sum=0
while a>0:
  r=a%10
  sum=sum+r
  a=a//10
if b%sum==0:
  print(b,"is a Harshad number")
else:
  print(b,"is not a Harshad number")
